src/features/unmarkedWsNoticeList/utils.ts

```diff
@@ -0,0 +1,78 @@
+import { ApplicationStatus } from '../../@types/__generated__/globalTypes';
+import {
+  UNMARKED_WINTER_STORAGE_NOTICES,
+  UNMARKED_WINTER_STORAGE_NOTICES_winterStorageNotices_edges as UNMARKED_WINTER_STORAGE_NOTICES_EDGE,
+  UNMARKED_WINTER_STORAGE_NOTICES_winterStorageNotices_edges_node as UNMARKED_WINTER_STORAGE_NOTICES_NODE,
+} from './__generated__/UNMARKED_WINTER_STORAGE_NOTICES';
+
+interface UnmarkedWinterStorageChoice {
+  winterStorageAreaName: string;
+  winterStorageArea: string;
+}
+
+export type UnmarkedWinterStorageNotice = {
+  boatLength: number;
+  boatModel: string;
+  boatName: string;
+  boatRegistrationNumber: string;
+  boatType?: string | null;
+  boatWidth: number;
+  choice: UnmarkedWinterStorageChoice;
+  createdAt: string;
+  customerId?: string;
+  firstName: string;
+  id: string;
+  lastName: string;
+  status: ApplicationStatus;
+};
+
+export const getUnmarkedWinterStorageNotices = (
+  data: UNMARKED_WINTER_STORAGE_NOTICES | undefined
+): UnmarkedWinterStorageNotice[] => {
+  return (
+    data?.winterStorageNotices?.edges.reduce<UnmarkedWinterStorageNotice[]>((acc, edge) => {
+      const {
+        boatLength,
+        boatModel,
+        boatName,
+        boatRegistrationNumber,
+        boatType,
+        boatWidth,
+        createdAt,
+        firstName,
+        id,
+        lastName,
+        status,
+        winterStorageAreaChoices: choices,
+      } = (edge as UNMARKED_WINTER_STORAGE_NOTICES_EDGE).node as UNMARKED_WINTER_STORAGE_NOTICES_NODE;
+
+      const choice: UnmarkedWinterStorageChoice =
```

Comment:
There seems to be duplicate logic in `src/features/unmarkedWsNoticeView/utils.ts` we could extract util fn?

Reply:
Sure! Since it's only used by these two features, I'll export it from one feature and import in another.

I also noticed some small issues with the utils, making a fix commit now.